pom.xml

```diff
@@ -37,14 +37,14 @@
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <python.ver>python &gt;= 2.6</python.ver>
     <deb.python.ver>python (&gt;= 2.6)</deb.python.ver>
-    <!--TODO change to HDP URL-->
+    <slf4j.version>2.0.0</slf4j.version>
     <hbase.tar>https://archive.apache.org/dist/hbase/2.4.2/hbase-2.4.2-bin.tar.gz</hbase.tar>
     <hbase.folder>hbase-2.4.2</hbase.folder>
     <hadoop.tar>https://archive.apache.org/dist/hadoop/common/hadoop-3.1.1/hadoop-3.1.1.tar.gz</hadoop.tar>
     <hadoop.folder>hadoop-3.1.1</hadoop.folder>
     <hadoop.version>3.1.1</hadoop.version>
-    <grafana.folder>grafana-5.2.4</grafana.folder>
-    <grafana.tar>https://s3-us-west-2.amazonaws.com/grafana-releases/release/grafana-5.2.4.linux-amd64.tar.gz</grafana.tar>
+    <grafana.folder>grafana-6.7.4</grafana.folder>
+    <grafana.tar>https://dl.grafana.com/oss/release/grafana-6.7.4.linux-amd64.tar.gz</grafana.tar>
```

Comment:
I am not aware of how we can fetch grafana tarball, other than that, changes look good